### PR TITLE
Update setup.py to include Changelog path for PyPI

### PR DESCRIPTION
**Pull request by rajulkumar**
PyPI uses the Changelog path in setup.py to show the Changelog
on the portal. So added the path in setup.py along with other
relevant information about the package. Also, modified the
Changelog.md to hyperlink the list of commits in a specific
version.

### CODE REVIEW

**Comment by rajulkumar** [comment]
No. This is just a good to have enhancement identified during the release.

File: setup.py [hunk 1]
```diff
@@ -31,16 +31,22 @@ def get_long_description():
...
       long_description_content_type="text/markdown",
       classifiers=[
+            "Development Status :: 4 - Beta",
+            "Intended Audience :: Developers",
             "Programming Language :: Python :: 2",
             "Programming Language :: Python :: 2.4",
             "Programming Language :: Python :: 2.6",
             "Programming Language :: Python :: 2.7",
             "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+            "Topic :: Software Development :: Libraries :: Python Modules",
```
Comment (rohanpm) [inline setup.py:41]: This isn't accurate, alt-src isn't intended for use as a library.

Comment (rajulkumar) [reply setup.py:41]: I saw this being used as a dependency to Pub. So should that be put out as a tool?
If so, should this be categorized as Software Development :: Version Control :: Git OR System :: Software Distribution OR Utilities

Comment (rohanpm) [reply setup.py:41]: "Software Development :: Version Control :: Git" makes sense to me, but I don't think you really need to add anything here.  I just didn't want to see it falsely referred to as a library.

Comment (rajulkumar) [reply setup.py]: Removed the Topic.

**Pull request merged**